src/main.ts

```diff
@@ -82,4 +101,14 @@ async function run(): Promise<void> {
   }
 }
 
+async function replaceAsync(str, regex, asyncFn) {
+  const promises = [] as any
+  str.replace(regex, (match, ...args) => {
+    const promise = asyncFn(args[0])
```

Comment:
Is there a reason that match is omitted and we're only passing the first arg to asyncFn?

Reply:
The match includes the brackets where as the first arg references the capture group of the regex which is just the issue number.